src/Aspire.Dashboard/Components/Controls/SelectResourceOptions.razor.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Concurrent;
+
+namespace Aspire.Dashboard.Components;
+
+public partial class SelectResourceOptions<TValue>
+{
+    private async Task OnAllValuesCheckedChangedInternalAsync(bool? newAreAllVisible)
+    {
+        SetCheckState(newAreAllVisible, AllValues);
+        await OnAllResourceTypesCheckedChangedAsync();
+    }
```

Comment:
```suggestion
    private async Task OnAllValuesCheckedChangedInternalAsync(bool? newAreAllVisible)
    {
        SetCheckState(newAreAllVisible, Values);
        await OnResourceTypesCheckedChangedAsync();
    }
```
Don't need `All` in many of these names anymore (I think this suggestion is right). All should just be used when working with the all button now.